tools/pythonpkg/tests/fast/spark/test_replace_empty_value.py

```diff
@@ -17,37 +17,46 @@ def test_replace_empty(self, spark):
 
         # Replace name
         # CASE WHEN "name" == '' THEN NULL ELSE "name" END
-        from duckdb.experimental.spark.sql.functions import col, when
+        from spark_namespace.sql.functions import col, when
 
         df2 = df.withColumn("name", when(col("name") == "", None).otherwise(col("name")))
         assert df2.columns == ['name', 'state']
         res = df2.select('name').collect()
         assert res == [Row(name=None), Row(name='Julia'), Row(name='Robert'), Row(name=None)]
 
         # Replace state + name
-        from duckdb.experimental.spark.sql.functions import col, when
+        from spark_namespace.sql.functions import col, when
 
         df2 = df.select([when(col(c) == "", None).otherwise(col(c)).alias(c) for c in df.columns])
```

Comment:
Should we not add a `sort` to this call instead of sorting the result manually afterwards?

Reply:
Good that you bring it up. I now switched to `sort` and it shows that Spark has a null-first rule in sorting and duckdb null-last. I've added a `FIXME`.